bridge/cypress/support/pageobjects/SequencesPage.ts

```diff
@@ -243,10 +243,10 @@ export class SequencesPage {
     return this;
   }
 
-  public assertIsSelectedSequenceWaiting(status: boolean): this {
+  public assertDtAlert(status: boolean): this {
```

Comment:
Could you make it more clear in the function name what it does?
```suggestion
  public assertDtAlertExists(status: boolean): this {
```